apigee_edge.links.menu.yml

```diff
@@ -44,8 +44,9 @@ apigee_edge.product:
   route_name: apigee_edge.settings.product.alias
   weight: -2
 
-apigee_edge.my_apps:
-  class: Drupal\apigee_edge\Plugin\Menu\MyAppsMenuLink
-  route_name: apigee_edge.user.my_apps
+apigee_edge.apps:
+  title: 'Apps'
+  description: 'A developers apps.'
```

Comment:
s/developer/developer's/g 
Or "Apps of a developer" if there is an issue with using `'`
Or leave it without description as it was.